rioting-daisies/BangazonWorkforce
# Jh edit employee

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Made the edit view so user can change first and last name as well as select a department and select an available computer as well as switch from one computer to another computer.  



Fixes Issue # https://github.com/rioting-daisies/BangazonWorkforce/issues/8

## Type of change

Please delete options that are not relevant.



# Testing Instructions

git fetch --all
git checkout jh-editEmployee
Check to make sure you can edit the employee's first name
Check to make sure you can edit the employee's last name
Check to make sure you can assign the employee to a different department
Check to make sure you can assign the employee a new computer

# Checklist:
